spec/acceptance/nodesets/fedora21.yml

```diff
@@ -2,9 +2,9 @@ HOSTS:
   master:
     roles:
       - master
-    platform: fedora-20-x86_64
+    platform: fedora-21-x86_64
```

Comment:
Good call, I'll bump it. It was originally configured to use fedora 20 so chances are the jenkins job is doing this too :\ I'll take a look.